### PR TITLE
Changes to work with new pxf_infra repo

### PR DESCRIPTION
**Pull request by shivzone**
PXF testing related repositories (pxf_automation, parot and singlecluster) have been combined into a single repo [pxf_infra](https://github.com/Pivotal-DataFabric/pxf_infra). The keys have already been added to [gpdb_common-ci-secrets](https://github.com/greenplum-db/continuous-integration/blob/master/secrets/gpdb_common-ci-secrets.yml). Updated the gpdb pipeline to use the new repo.
@pivotal-mike @divyabhargov @lavjain This is a fairly simple change but still need a quick review

### CODE REVIEW

**Comment by shivzone** [comment]
@pivotal-mike I've rebased the branch

**Comment by shivzone** [comment]
@pivotal-mike good catch. I've updated it to work with the chown fix path fix.

**Comment by shivzone** [comment]
@edespino Need your review as well since you introduced the gpdb pipeline templates.

**Comment by edespino** [comment]
The master pipeline doesn't appear to have been generated with gen_pipeline.py. I noticed this because the timestamp isn't shown as a diff.

**Comment by shivzone** [comment]
@edespino you are right. Its fixed now

File: concourse/pipelines/gpdb_master-generated.yml [hunk 1]
```diff
@@ -3013,14 +3015,14 @@ jobs:
...
-    - get: pxf_automation_src
+    - get: pxf_infra_src
       trigger: true
     - get: centos-gpdb-dev-6
   - task: regression_tests_pxf
     file: gpdb_src/concourse/tasks/regression_tests_pxf.yml
     image: centos-gpdb-dev-6
     params:
-      GROUP: {{pxf_automation_test_group}}
+      GROUP: {{pxf_infra_test_group}}
```
Comment (edespino) [inline concourse/pipelines/gpdb_master-generated.yml]: Can you confirm the **GROUP** variable is used by concourse/scripts/regression_tests_pxf.bash? I couldn't spot how it is used.

**Comment by edespino** [comment]
- I was able to run **gen_pipeline.py -t prod** to confirm the output matches (excluding the timestamp) the **gpdb_master-generated.yml** file in this review.
- I set the pipeline (**gpdb_master-generated.yml** - included in this review) successfully to ensure all resources are used and resolve properly.
- I validated the referenced and newly added external variables have been added to **gpdb_common-ci-secrets.yml**
- I reviewed the flow of the newly named resources

**Comment by edespino** [comment]
Aside from the outstanding **GROUP** question, **+1** to this PR.

File: concourse/pipelines/gpdb_master-generated.yml (hunk 1 above)
Comment (shivzone) [reply concourse/pipelines/gpdb_master-generated.yml]: I suppose GROUP param was introduce to configure ```make GROUP=gpdb``` in run_pxf_automation in concourse/scripts/regression_tests_pxf.bash. I can update it accordingly.

Comment (edespino) [reply concourse/pipelines/gpdb_master-generated.yml]: As you will have two distinct pipelines, the pipelines themselves can contain the pxf test configuration. If that is the case, there is no need to put it into the credential file.

**Pull request closed without merging**